packages/@react-spectrum/table/test/TableDnd.test.js

```diff
@@ -35,7 +35,9 @@ import {useDragAndDrop} from '@react-spectrum/dnd';
 import {useListData} from '@react-stately/data';
 import userEvent from '@testing-library/user-event';
 
-let isReact18 = parseInt(React.version, 10) >= 18;
+// getComputedStyle is very slow in our version of jsdom.
+// These tests only care about direct inline styles. We can avoid parsing other stylesheets.
+window.getComputedStyle = (el) => el.style;
```

Comment:
didn't want to do in a beforeAll using `jest.spyOn(window, 'getComputedStyle').mockImplementation(el => el.style)`?